### PR TITLE
Refactor portal header

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

Greatly simplify the structure of the `Header` component. It seems like the `Card` component from Jøkul was used as a basis, and a lot of the props/functionality from that was still implemented.

I realize I've also made it a bit less generalized, but I don't think we need to implement patterns for headers we don't really _have_ a pattern for yet.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

This is not directly related to this PR, but I'm not sure that i like this architecture:

Skjermbilde 2020-02-19 kl  09 34 33

I presume it is meant to approximate the structure of the packages in Jøkul, but is this the right place to do that? If we assume that our code will be "stolen" by the teams, we should write the portal and its architecture in the way we would like the teams to write their projects, not as a library.

As it stands now, we get a lot of folders, winding paths, and not-so-pretty imports like this:

Skjermbilde 2020-02-19 kl  09 51 45

The packages in Jøkul (which you will automatically have available if you edit the portal) already illustrate the architecture of the library, so shouldn't we rather illustrate the architecture we proposed to the teams earlier?:

Skjermbilde 2020-02-19 kl  09 55 50

**Edit**: This could also be a great teaching exercise for a Jøkul bootcamp/hackathon: refactoring a component from a project (the portal?) to a Jøkul library style- and React package.

### CODE REVIEW

File: portal/src/components/Header/header-react/src/header.tsx [hunk 1]
```diff
@@ -1,53 +1,15 @@
...
-        </header>
-    );
-};
+export const Header = () => (
+    <header className="portal-header">
+        <Link to="/" className="portal-header__title">
+            <strong>Jøkul</strong> Designsystem
+        </Link>
+        <div className="portal-header__logo">
+            <Logo isSymbol centered={false} />
```
Comment (lfbergee) [inline portal/src/components/Header/header-react/src/header.tsx]: Burde vi ikke animere?

Comment (piofinn) [reply portal/src/components/Header/header-react/src/header.tsx]: Det funker ikke så bra når den er plassert til høyre.. Men er enig i at det hadde vært fornuftig å vise frem! Får ta en runde med design på det (Joachim jobber uansett med nye skisser virker det som).

Comment (lfbergee) [reply portal/src/components/Header/header-react/src/header.tsx]: fixed header med 120% height på window.scrollY(0,0), 100% height med symbol ellers

**Comment by lfbergee** [comment]
Skjønner ikke helt hva den further comments peker til. 

Vi har ikke den strukturen i portalen i dag? Anyways, ref Dan Abramov https://mobile.twitter.com/dan_abramov/status/1027248875072114689 evt http://react-file-structure.surge.sh/

Men ja, er enig med det at samlokasjon på intensjon er bedre enn å samle basert på funksjon eller teknologi.

**Comment by piofinn** [comment]
> Vi har ikke den strukturen i portalen i dag?

Denne PR-en er ikke mot `master`, men mot branchen der @eduardodir og @joms har jobbet litt med refaktorering og redesign av portalen. Tok det som en PR for å ha et sted å diskutere arkitekturen i forbindelse med en relevant kodeendring.

**Pull request merged**